content/src/service/ServiceImpl.ts

```diff
@@ -363,16 +363,21 @@ export class ServiceImpl implements MetaverseContentService, ClusterDeploymentsS
   }
 
   getDeployments(options?: DeploymentOptions, task?: Database): Promise<PartialDeploymentHistory<Deployment>> {
-    return this.repository.reuseIfPresent(task, (db) =>
-      db.taskIf((task) =>
-        this.deploymentManager.getDeployments(task.deployments, task.content, task.migrationData, options)
-      )
+    return this.repository.reuseIfPresent(
+      task,
+      (db) =>
+        db.taskIf((task) =>
+          this.deploymentManager.getDeployments(task.deployments, task.content, task.migrationData, options)
+        ),
+      { priority: DB_REQUEST_PRIORITY.HIGH }
```

Comment:
I'm not sure that this change should be together with the metrics. Also, are there any requests that are of LOW priority with this change?